src/query/graphite/native/builtin_functions_test.go

```diff
@@ -1225,6 +1345,8 @@ func TestLowestAverage(t *testing.T) {
 	testRanking(t, ctx, tests, lowestAverage)
 }
 
+
+
```

Comment:
gofmt would make this so it's just a single end liine between functions. Could you fix that? Not sure why our linter isn't picking this up..

Reply:
Fixed by running go fmt